common/src/Model/Brush.cpp

```diff
@@ -880,7 +880,7 @@ namespace TrenchBroom {
             return updateGeometryFromFaces(worldBounds);
         }
 
-        kdl::result<std::vector<Brush>, BrushError> Brush::subtract(const MapFormat mapFormat, const vm::bbox3& worldBounds, const std::string& defaultTextureName, const std::vector<const Brush*>& subtrahends) const {
+        std::tuple<std::vector<Brush>, std::optional<BrushError>> Brush::subtract(const MapFormat mapFormat, const vm::bbox3& worldBounds, const std::string& defaultTextureName, const std::vector<const Brush*>& subtrahends) const {
```

Comment:
I would have expected this function to return a vector of results now.

Reply:
I followed this idea and pushed a fixup commit: 7cb9cac1e -- feel free to drop if you don't like it, but, having worked with the result type a lot, the code feels a little more natural to me.

Reply:
Nice, I tried that as well on an earlier iteration but didn't have the collect_values idea. Let's go with that.